<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowHistogramPlot</class>
 <widget class="QMainWindow" name="WindowHistogramPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>567</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>UpArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Histogram</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="HistogramPlot" name="qwtPlotHistogram">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="canvasBackground" stdset="0">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>Configure</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionClear"/>
    <addaction name="actionDuplicate"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionSaveImage"/>
    <addaction name="actionSaveData"/>
    <addaction name="actionSaveDataHEPRO"/>
   </widget>
   <widget class="QMenu" name="menuFitting">
    <property name="title">
     <string>Fitting</string>
    </property>
    <addaction name="actionGaussian"/>
    <addaction name="actionDoubleGaussian"/>
   </widget>
   <widget class="QMenu" name="menuCuts">
    <property name="title">
     <string>Cuts</string>
    </property>
    <widget class="QMenu" name="menuLinear">
     <property name="title">
      <string>Linear</string>
     </property>
     <addaction name="actionAddLinearCut"/>
     <addaction name="actionEditLinearCut"/>
     <addaction name="actionRemoveLinearCut"/>
    </widget>
    <addaction name="menuLinear"/>
    <addaction name="separator"/>
    <addaction name="actionActiveCuts"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLogarithmicScale"/>
    <addaction name="actionDeltaHistogram"/>
    <addaction name="separator"/>
    <addaction name="actionRename_window"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfigure"/>
   <addaction name="menuCuts"/>
   <addaction name="menuFitting"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="shortcut">
    <string>C, O</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Save image</string>
   </property>
   <property name="shortcut">
    <string>P, I</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="text">
    <string>Save data</string>
   </property>
   <property name="shortcut">
    <string>P, D</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>C, R</string>
   </property>
  </action>
  <action name="actionActiveCuts">
   <property name="text">
    <string>Active cuts</string>
   </property>
   <property name="shortcut">
    <string>C, A</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>C, C</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>Gaussian</string>
   </property>
   <property name="shortcut">
    <string>F, G</string>
   </property>
  </action>
  <action name="actionDoubleGaussian">
   <property name="text">
    <string>Double Gaussian</string>
   </property>
   <property name="shortcut">
    <string>F, D</string>
   </property>
  </action>
  <action name="actionAddLinearCut">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>L, A</string>
   </property>
  </action>
  <action name="actionRemoveLinearCut">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>L, R</string>
   </property>
  </action>
  <action name="actionEditLinearCut">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>L, E</string>
   </property>
  </action>
  <action name="actionLogarithmicScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Logarithmic scale</string>
   </property>
   <property name="shortcut">
    <string>V, L</string>
   </property>
  </action>
  <action name="actionDeltaHistogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delta histogram</string>
   </property>
   <property name="shortcut">
    <string>V, D</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="shortcut">
    <string>C, D</string>
   </property>
  </action>
  <action name="actionRename_window">
   <property name="text">
    <string>Rename window</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionSaveDataHEPRO">
   <property name="text">
    <string>Save data (HEPRO)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HistogramPlot</class>
   <extends>QwtPlot</extends>
   <header>HistogramPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onCloseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveData</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onSaveDataClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveImage</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onSaveImageClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onRefreshClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOptions</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onOptionsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionActiveCuts</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onActiveCutsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>clearValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGaussian</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onFitGaussianClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDoubleGaussian</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onFitDoubleGaussianClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeltaHistogram</sender>
   <signal>toggled(bool)</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onDeltaHistogramToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLogarithmicScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onLogScaleToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDuplicate</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onDuplicateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRename_window</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onRenameClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddLinearCut</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onAddLinearCutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditLinearCut</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onEditLinearCutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemoveLinearCut</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onRemoveLinearCutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveDataHEPRO</sender>
   <signal>triggered()</signal>
   <receiver>WindowHistogramPlot</receiver>
   <slot>onSaveDataHEPROClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSaveDataClicked()</slot>
  <slot>onSaveImageClicked()</slot>
  <slot>onRefreshClicked()</slot>
  <slot>onOptionsClicked()</slot>
  <slot>onActiveCutsClicked()</slot>
  <slot>clearValues()</slot>
  <slot>onFitGaussianClicked()</slot>
  <slot>onFitDoubleGaussianClicked()</slot>
  <slot>onLogScaleToggled(bool)</slot>
  <slot>onDeltaHistogramToggled(bool)</slot>
  <slot>onDuplicateClicked()</slot>
  <slot>onRenameClicked()</slot>
  <slot>onAddLinearCutClicked()</slot>
  <slot>onEditLinearCutClicked()</slot>
  <slot>onRemoveLinearCutClicked()</slot>
  <slot>onSaveDataHEPROClicked()</slot>
 </slots>
</ui>
